src/oic/oauth2/message.py

```diff
@@ -1100,7 +1101,42 @@ class ASConfigurationResponse(Message):
 
 
 def factory(msgtype):
+    warnings.warn('`factory` is deprecated. Use `OIDCMessageFactory` instead.', DeprecationWarning)
```

Comment:
Looks like the wrong warning for the oauth2 module.

Reply:
Yep, copy paste error...